includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -199,6 +199,7 @@ public function enqueue_scripts() {
 			array(
 				'pending_jetpack_install' => $pending_jetpack ? 'yes' : 'no',
 				'states'                  => WC()->countries->get_states(),
+				'postcodes'               => wp_list_pluck( WC()->countries->get_country_locale(), 'postcode' ),
```

Comment:
This throw several PHP notices like:

> PHP Notice:  Undefined index: postcode in wp-includes/class-wp-list-util.php on line 154

This happens because not all items inside the `WC()->countries->get_country_locale()` contains `postcode`.

Reply:
Thanks for review, I missed this. Refactored to use a helper method instead.